### PR TITLE
Feat 5389 - When a new account is created or admin har reset password send email to user

### PR DESCRIPTION
**Pull request by xmacan**
done #5389 

Adding log message too

### CODE REVIEW

**Comment by xmacan** [comment]
@TheWitness - It is done, please check

**Comment by TheWitness** [comment]
Will do.

File: include/global_settings.php [hunk 1]
```diff
@@ -1968,6 +1968,70 @@
...
+	'secnotify_chpass_message' => array(
+		'friendly_name' => __('Email body for reset password message'),
+		'description'   => __('This is the message that will be send when reset user password request. Max 1024 characters. HTML is allowed. There are several common replacement tags that may be used in include &#060CACTIURL&#062 &#060USERNAME&#062 &#060PASSWORD&#062'),
+		'method'        => 'textarea',
+		'textarea_rows' => '4',
+		'textarea_cols' => '80',
+		'max_length'    => 1024,
+		'default'       => __('User\'s password has been reset. <br>Server: <a href="<CACTIURL>"><CACTIURL></a><br>Username: <USERNAME><br>To set a new password, use <a href="<PWDRESETLINK>">the following link</a><br><br>')
+	),
+	'secnotify_resetlink_timeout' => array(
```
Comment (TheWitness) [inline include/global_settings.php:2050]: This setting is normally not more than say 15 minutes.  So, I would have:

5 Minutes
10 Minutes
15 Minutes
20 Minutes.

**Review by TheWitness — CHANGES_REQUESTED** [review]

**Comment by TheWitness** [comment]
@xmacan, did you see my request for change?

**Comment by xmacan** [comment]
yes, I will do it asap

File: include/global_settings.php (hunk 1 above)
Comment (xmacan) [reply include/global_settings.php:2050]: done

**Review by TheWitness — APPROVED** [review]

**Pull request merged**